src/main/java/zmaster587/advancedRocketry/item/components/ItemJetpack.java

```diff
@@ -31,13 +31,14 @@
 import zmaster587.libVulpes.util.FluidUtils;
 import zmaster587.libVulpes.util.InputSyncHandler;
 
+import javax.annotation.Nonnull;
 import java.util.List;
 
 public class ItemJetpack extends Item implements IArmorComponent, IJetPack {
 	
-	private static enum MODES {
```

Comment:
Any reason why this became nonstatic?

Reply:
The static modifier is redundant for inner enums.

Reply:
Ah, forgot that